d2/src/test/java/com/linkedin/d2/balancer/servers/ZookeeperConnectionManagerTest.java

```diff
@@ -437,7 +438,7 @@ public void testMarkUpAndDownMultipleTimesFinalDown()
     executorService.shutdown();
   }
 
-  @Test(invocationCount = 10, timeOut = 10000)
+  @Test(invocationCount = 10, timeOut = 10000, groups = { "ci-flaky" })
```

Comment:
what does this group do? Is there a way to disable all ci-flaky tests after you group them?

Reply:
All tests in the group `ci-flaky` are conditionally excluded if being run in Travis. If being run locally, they will still be run. I added logic for this in the root `build.gradle`, search for the reference to `ci-flaky`